src/archutils/Win32/ddk/hidpi.h

```diff
@@ -306,8 +306,8 @@ Routine Description:
    Capabilities     a HIDP_CAPS structure
 
 Return Value:
-�  HIDP_STATUS_SUCCESS
-�  HIDP_STATUS_INVALID_PREPARSED_DATA
+�  HIDP_STATUS_SUCCESS
```

Comment:
possible encoding issues in this file?

Reply:
This file was originally in the codebase with ISO-8859-1 encoding. The mojibake was caused by a bad conversion to UTF-8. I downloaded the previous file before I touched it, converted it to UTF-8, and the fixed version will be in the next commit.